bin/auto_hck

```diff
@@ -11,6 +11,8 @@ begin
   require './lib/project'
   require './lib/trap'
 
+  exec File.join(__dir__, 'ns'), __FILE__, *ARGV unless ENV.key?('AUTOHCK_SLIRP')
```

Comment:
Is this safe to start something via exec?

Reply:
I think so. We already have multiple `spawn` calls, which should include `exec`.